### PR TITLE
Correct README

### PR DESCRIPTION
**Pull request by yanagi0324**
I found out that **Add to your existing project** section is not all correct. Because don't mention about `children_count`. I added it to the section.

I read **guidelines for contributing**. Although I thought I don't need to write **CHANGELOG** for this change, if I need it, please tell me.

Thanks reading this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ end
 
 To make use of `awesome_nested_set`, your model needs to have 3 fields:
 `lft`, `rgt`, and `parent_id`. The names of these fields are configurable.
-You can also have an optional field, `depth`.
+You can also have optional fields, `depth` adn `children_count`.
```
Comment (parndt) [inline README.md]: You have a typo: 'adn' instead of 'and' :simple_smile:

Comment (yanagi0324) [inline README.md]: Oops. I fixed it. Thanks  :smile:

**Comment by yanagi0324** [comment]
@parndt Hi, when will this PR be merged or deleted due to bad PR? What should I do?

**Comment by parndt** [comment]
Thank you!

**Pull request merged**